Create.java

```diff
@@ -1,15 +1,183 @@
 import java.sql.Connection;
 import java.sql.ResultSet;
 import java.sql.Statement;
-public class Create {
+import java.sql.Connection;
+import java.sql.ResultSet;
+
```

Comment:
Need to import scanner.  I ran this on my machine and there was an error because there was no scanner